improver/cli/apply_emos_coefficients.py

```diff
@@ -137,7 +145,7 @@ def process(
 
     from improver.calibration.ensemble_calibration import ApplyEMOS
 
-    if cube.name() in ["emos_coefficients", "land_binary_mask"]:
+    if cube.name() == "land_binary_mask":
```

Comment:
I think we can drop this check. The plugin now checks that the diagnostic being calibrated has a name that matches those used to construct the coefficients file; it does this early. As such there is no risk of us calibrating the land mask :-)

Reply:
I’ve removed this check.